testdata/p4tc_samples_outputs/add_entry_1_example_control_blocks.c

```diff
@@ -108,7 +109,7 @@ if (/* hdr->ipv4.isValid() */
                                         .tblid = 1,
                                         .profile_id = 2
                                     };
-                                    bpf_p4tc_entry_create_on_miss(skb, &update_params, &key, sizeof(key), &update_act_bpf);
+                                    bpf_p4tc_entry_create_on_miss(skb, &update_params, sizeof(params), &key, sizeof(key));
```

Comment:
Sorry for the late review, but didn't have time to test before it was merged

Should be:
```
bpf_p4tc_entry_create_on_miss(skb, &update_params, sizeof(update_params), &key, sizeof(key));
```